module-4/backend/week_one.md

```diff
@@ -5,16 +5,31 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. What's the most useful thing you learned from completing the intermission week work?
+Merge Sort
 2. What are some tools to help debug JavaScript code?
+Chrome dev tools, pryjs, console.log
 3. What are some tools you need in order to unit test your JavaScript?
+Mocha, Chai.assert
 4. What is the syntax for invoking a function?
+func();
 5. What's the difference between `==` and `===` in JavaScript?
+=== checks data type and value (To return true, both sides must be strings, or ints, or whatever in addition to having the same values)
+== checks for same values, but not same datatype (So '0' == 0 returns true)
 6. What's the difference between asynchronous and synchronous JavaScript? 
+Synchronous JS is 'blocking', meaning every line of code must finish executing before the next line can execute. Asynchronous is more flexible and allows code to be triggered after certain events via callbacks
 7. How do we setup a route when creating an API with Node and Express?
+app.get('path', (request, response) => response.send() );
 8. What do we use Knex for?
+Interacting with the database
 9. What's `npm` and what do we use it for?
+Node Package Manager. It handles our project's dependencies.
 
 #### Review  
 10. What's the MVC design pattern? Describe each part of MVC?
+Model - Interfaces with DB
+View - Creates the HTML documents for the browser
+Controller - Interfaces b/t model, view and routes.
 11. What is AJAX? What are some benefits of using AJAX?
+Ajax is a strategy to fetch info from a server w/o initiating a complete http request/response cycle
```

Comment:
Great! I think it's still technically a complete req/resp cycle, just occurring in the background.